integration-tests/test/native-eth.spec.ts

```diff
@@ -45,21 +45,21 @@ describe('Native ETH Integration Tests', async () => {
       const amount = utils.parseEther('0.5')
       const addr = '0x' + '1234'.repeat(10)
       const gas = await env.ovmEth.estimateGas.transfer(addr, amount)
-      expect(gas).to.be.deep.eq(BigNumber.from(17344))
+      expect(gas).to.be.deep.eq(BigNumber.from(0x43c0)) // 17344
```

Comment:
Setting the value in hex makes it easier to update these values when they change.